esmf_regrid/tests/unit/experimental/unstructured_scheme/test__create_cube.py

```diff
@@ -11,17 +13,56 @@ def test_create_cube_2D():
     data = np.ones([2, 3])
 
     # Create a source cube with metadata and scalar coords
-    src_cube = iris.cube.Cube(np.zeros(5))
+    src_cube = Cube(np.zeros(5))
     src_cube.units = "K"
     src_cube.attributes = {"a": 1}
     src_cube.standard_name = "air_temperature"
-    scalar_height = iris.coords.AuxCoord([5], units="m", standard_name="height")
-    scalar_time = iris.coords.DimCoord([10], units="s", standard_name="time")
+    scalar_height = AuxCoord([5], units="m", standard_name="height")
+    scalar_time = DimCoord([10], units="s", standard_name="time")
     src_cube.add_aux_coord(scalar_height)
     src_cube.add_aux_coord(scalar_time)
 
     mesh_dim = 0
 
+    grid_x = DimCoord(np.arange(3), standard_name="longitude")
+    grid_y = DimCoord(np.arange(2), standard_name="latitude")
+
+    cube = _create_cube(data, src_cube, mesh_dim, grid_x, grid_y)
+    src_metadata = src_cube.metadata
```

Comment:
should `_create_cube` be handling metadata as well?

Reply:
`_create_cube` already handles metadata, this is also the case in iris.

Reply:
Ah yes, sorry I misread this section.  You're getting the source metadata to attach to the `expected_cube`, not to `cube`.